test/e2e/framework/service/jig.go

```diff
@@ -962,3 +962,18 @@ func (j *TestJig) CreateTCPUDPServicePods(replica int) error {
 	}
 	return e2erc.RunRC(config)
 }
+
+// CreateSCTPServiceWithPort creates a new SCTP Service with given port based on the
+// j's defaults. Callers can provide a function to tweak the Service object before
+// it is created.
+func (j *TestJig) CreateSCTPServiceWithPort(tweak func(svc *v1.Service), port int32) (*v1.Service, error) {
```

Comment:
Which e2es are consuming this? Or are there external consumers vendoring the e2e framework?

Reply:
@andrewsykim These would be the e2e cases: https://github.com/kubernetes/kubernetes/pull/88196